extensions/amp-story-captions/0.1/test/test-amp-story-captions.js

```diff
@@ -80,6 +75,16 @@ describes.realWin(
       expect(element.querySelector('div').textContent).to.equal(
         'first caption'
       );
+      return video;
+    }
+
+    it('should contain be empty when built', async () => {
```

Comment:
Nit: `container` you meant?

Reply:
Actually didn't touch this test, I'm not sure why git has it as a changed line.
Definitely a typo though I'll fix it :)